package.json

```diff
@@ -5,19 +5,19 @@
   "scripts": {
     "build": "NODE_OPTIONS='--max_old_space_size=4096' turbo run build",
     "check": "pnpm typecheck && pnpm lint && pnpm prettier:check",
-    "check:fix": "pnpm typecheck && pnpm lint:fix",
+    "check:fix": "pnpm typecheck; pnpm lint:fix",
```

Comment:
why this change? if `typecheck` fails you still want to run `pnpm lint:fix`? `&&` enforce to stop immediately command and return exit code 1, `;` doesn't return exit code at all

```json
"lint": "eslint --cache --ignore-path .gitignore . && echo hello",
```
<img width="398" alt="image" src="https://github.com/graphprotocol/docs/assets/7361780/6d33cec4-5bba-4180-b016-4cc79d608bb6">

```json
"lint": "eslint --cache --ignore-path .gitignore . ; echo hello",
```

<img width="249" alt="image" src="https://github.com/graphprotocol/docs/assets/7361780/79228300-9085-49b2-93a4-2594958e7878">

Reply:
@B2o5T – My reasoning is that the `check` script should fail _as early as possible_, since it's used in CI to check if everything's good to go or if there's an issue (any issue at all). Whereas the `check:fix` script is meant to be run locally (I personally run it before every commit), and it should list _all_ the issues and fix as many of them as possible. For instance, I don't feel that an ESLint error should be a reason not to format the code using Prettier. And since we only run it locally, `check:fix` doesn't need to return an error code. I suppose it wouldn't hurt if it did, but I'm not sure if there's an easy way to do that while not stopping early if the first process returns a non-zero exit code?

Reply:
got it, lgtm